frontend/src/firebase/firebaseInteractor.ts

```diff
@@ -130,9 +130,17 @@ export default class FirebaseInteractor {
         currentInterventionOrAssessment:
           userData.currentInterventionOrAssessment || "oiBN8aE5tqEFK2gXJUpl",
       };
+    } else {
+      throw new Error("Invalid user");
     }
   }
 
+  async createCurrentUser() {
+    try {
+      this.currentUser = await this.getUser(undefined);
+    } catch (error) {}
+  }
```

Comment:
maybe u can console out the error or throw an error here? since there's like nothing here for now

Reply:
Good point. Will do!